java_servlets/src/main/java/checkServlet.java

```diff
@@ -0,0 +1,21 @@
+import com.google.gson.Gson;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.io.PrintWriter;
+
+@WebServlet("/check")
+public class checkServlet extends HttpServlet {
```

Comment:
In Java, class names are by a very-very old and gold convention named from`CapitalLetter`. Unlike methods. Please make corrections